src/main/java/StressMain.java

```diff
@@ -649,6 +655,54 @@ private static Callable taskCallable(Workflow workflow, SolrCloud cloud, Map<Str
 				} catch (Exception ex) {
 					ex.printStackTrace();
 				}
+			} else if (type.moveReplica != null) {
+				long taskStart = System.currentTimeMillis();
+
+				String collectionName = params.get("COLLECTION");
```

Comment:
should we use `type.moveReplica.collection` here instead?

Reply:
To be honest, the `resolveString` stuff was just super confusing to me, and getting `COLLECTION` from the params just worked. But that should work, and it would give a purpose to `type.moveReplica.collection`, so I'll do that!